src/main/java/ch/njol/skript/log/LogEntry.java

```diff
@@ -130,7 +130,26 @@ public String toString() {
 		if (n == null || level.intValue() < Level.WARNING.intValue())
 			return message;
 		final Config c = n.getConfig();
-		return message + from + " (" + c.getFileName() + ", line " + n.getLine() + ": " + n.save().trim() + "')";
+		
+		String t = "    "; // Because \t shows unknown char in mc
+		boolean isError = level.intValue() == Level.SEVERE.intValue();
+		
+		String levelColor;
+		if (level.intValue() == Level.WARNING.intValue()) // Warnings
+			levelColor = "§e";
+		else if (isError) // Errors 
+			levelColor = "§c";
+		else // Anything else?
+			levelColor = "§f";
+		
+		return
```

Comment:
You should probably use a StringBuilder here

Reply:
Apology I didn't get an email of this comment, I will do them ASAP.